dm/pkg/schema/tracker.go

```diff
@@ -377,42 +377,42 @@ func (tr *Tracker) GetSystemVar(name string) (string, bool) {
 
 // GetDownStreamIndexInfo gets downstream PK/UK(not null) Index.
 // note. this function will init downstreamTrack's table info.
-func (tr *Tracker) GetDownStreamIndexInfo(tctx *tcontext.Context, tableID string, originTi *model.TableInfo) (*model.IndexInfo, error) {
+func (tr *Tracker) GetDownStreamIndexAndTableInfo(tctx *tcontext.Context, tableID string, originTi *model.TableInfo) (*model.IndexInfo, *DownstreamTableInfo, error) {
```

Comment:
```suggestion
func (tr *Tracker) GetDownStreamTableInfo(tctx *tcontext.Context, tableID string, originTi *model.TableInfo) (*DownstreamTableInfo, error) {
```

Reply:
seems only return `dit` is enough

Reply:
yes, i will fix `GetDownStreamIndexAndTableInfo` and its uts later.